### DISCUSSION TITLE
Translations updates for release

### DISCUSSION THREAD

**Discussion by stuarthayhurst** [stuarthayhurst/alphabetical-grid-extension#44] [Translations]
@Vistaus @daPhipz
New release coming soon, the translations files should be ready :)
Whenever you guys have time, I'd be awesome if you could update your translations. There might've been 2 extra fuzzy strings in one of the translation files I noticed, I can't really remember.

Thanks!

**Comment by Vistaus** [comment]
Done! #45

**Comment by daPhipz** [comment]
Same here! #46

**Comment by stuarthayhurst** [comment]
Wow, that was fast!
Thanks, both of you :D

**Comment by stuarthayhurst** [comment]
@daPhipz I'm going to do a maintenance release some point soon to tidy a few things up, do you think you could take a look at the 2 fuzzy strings in the French translations, whenever you have a chance?

Thanks :)

**Comment by daPhipz** [comment]
Done ;)

**Reply by stuarthayhurst** [reply]
Quick response as always, thank you :)
